x-pack/test/detection_engine_api_integration/security_and_spaces/group10/perform_bulk_action.ts

```diff
@@ -106,6 +106,11 @@ export default ({ getService }: FtrProviderContext): void => {
         missing_exception_lists_count: 0,
         missing_rules: [],
         missing_rules_count: 0,
+        excluded_action_connection_count: 0,
+        excluded_action_connections: [],
+        exported_action_connector_count: 0,
```

Comment:
Perhaps we should expand the one `BulkActionType.export` test to include actions and exceptions to ensure the proper clients are being plumbed through to `getExportByObjectIds()`?

Reply:
Definitely, thank you for pointing out! 
Will still add for this `export` and this file `x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/import_rules/route.ts` I still have one TODO 😄

Reply:
Please 🙏🏻 [validate](https://github.com/elastic/kibana/pull/148703/commits/31bcd730b7977074e3c52426d02433eb806b2569#diff-ca72cc2e18d47b583a9fe9f5d0f70557f31f094b3c46b13b6760a8beda3b8b79R116)